index.js

```diff
@@ -35,3 +35,35 @@ exports.verifyPayment = function (platform, payment, cb) {
 		cb(null, result);
 	});
 };
+
+
+exports.cancelSubscription = function (platform, payment, cb) {
+	function syncError(error) {
```

Comment:
I guess you meant `asyncError`?

Reply:
I created `cancelSubscription` by simply copying and adapting `verifyPayment`. I could improve both but maybe in a seperate PR?